petcare-sample/b2c/web-app/petdesk/web/react/src/components/GetThumbnail/get-thumbnail.ts

```diff
@@ -6,9 +6,15 @@ export async function getThumbnail(accessToken: string, petId: string) {
       'accept': `*/*`,
     };
 
+    try {
     const response = await getPetInstance().get("/pets/" + petId + "/thumbnail" , {
       headers: headers,
       responseType: 'blob',
     });
     return response;
-  }
\ No newline at end of file
+  } catch (error) {
+    // Handle errors here
+    console.error('Error fetching thumbnail:', error);
```

Comment:
Remove these logs.
Add a comment if you are making the errors silent.

Reply:
Removed with 566218d543bee0fe0230971d4e69d3566fa32f65.